README.rst

```diff
@@ -87,13 +87,16 @@ The easiest way to install ParMOO is via the Python package index, PyPI
 
 .. code-block:: bash
 
-    pip install [--user] parmoo
+    pip install < --user > parmoo
+
+where the braces around ``< --user >`` indicates that the ``--user`` flag is
```

Comment:
Consider:
where the braces around ``< --user >`` indicate that the ``--user`` flag is
(deletes an s)

Reply:
done